0-bootstrap/modules/jenkins/main.tf

```diff
@@ -112,7 +125,147 @@ resource "google_compute_firewall" "allow_ssh_to_jenkins_agent_fw" {
 
 resource "google_compute_network" "jenkins_agents" {
   project = module.cicd_project.project_id
-  name    = "jenkins-agents-network"
+  name    = "vpc-b-jenkinsagents"
 }
 
 // TODO(caleonardo): add an option to create a Jenkins Master in GCP, in case the user doesn't have one on-prem
+// If you do not have a Jenkins implementation and don't want one, you should use the Cloud Build module instead of this Jenkins module
+
+/******************************************
+  VPN Connectivity on-prem <--> CICD project
+*******************************************/
+
+// TODO(Amanda / Daniel): add VPN for connectivity between Jenkins Master on prem with Jenkins Agent in GCP
+
+/******************************************
+  Jenkins IAM for admins
+*******************************************/
+
+resource "google_project_iam_member" "org_admins_jenkins_admin" {
+  project = module.cicd_project.project_id
+  role    = "roles/compute.admin"
+  member  = "group:${var.group_org_admins}"
+}
+
+resource "google_project_iam_member" "org_admins_jenkins_viewer" {
+  project = module.cicd_project.project_id
+  role    = "roles/viewer"
+  member  = "group:${var.group_org_admins}"
+}
+
+/******************************************
+  Jenkins Artifact bucket
+*******************************************/
+
+resource "google_storage_bucket" "gcs_jenkins_artifacts" {
```

Comment:
what gcs artifacts are expected here?

Reply:
People can put here Jenkins build logs if they want to. It is optional. By default, any output from the Jenkins build will go to the GCE hard drive (`/home/jenkins`). Being optional, should we remove this then?

Reply:
I think this is a good to have, let keep it.